### PR TITLE
Add dbSNP b156 to resources for v4

### PR DESCRIPTION
**Pull request by KoalaQin**
Add dbSNP b156 to resources for gnomAD v4

### CODE REVIEW

**Comment by KoalaQin** [comment]
annotations.py is the changes Mike made with me in another PR, maybe you have to merge that one first.

**Comment by KoalaQin** [comment]
Sorry for so many commits, I was confused. Now finally only this file changed!

File: gnomad/resources/grch38/reference_data.py [hunk 1]
```diff
@@ -154,8 +154,22 @@ def _import_dbsnp(**kwargs) -> hl.Table:
 )
 
 dbsnp = VersionedTableResource(
-    default_version="b154",
+    default_version="b156",
     versions={
+        "b156": GnomadPublicTableResource(
+            # can't find this path anymore, is it temporary for intermediate files?
```
Comment (jkgoodrich) [inline gnomad/resources/grch38/reference_data.py:160]: Yeah it's temp for where It gets written to on import, then it gets synced to the GCP bucket. By default `GnomadPublicTableResource` will use the GCP bucket for loading.
```suggestion
```

File: gnomad/resources/grch38/reference_data.py [hunk 2]
```diff
@@ -154,8 +154,22 @@ def _import_dbsnp(**kwargs) -> hl.Table:
...
+            path="gs://gnomad-public-requester-pays/resources/grch38/dbsnp/dbsnp_b156_grch38_all_20221116.ht",
+            import_func=_import_dbsnp,
+            import_args={
+                "path": "gs://gcp-public-data--gnomad/resources/grch38/dbsnp/dbsnp_b156_grch38_all_GCF_000001405.40_20221116.vcf.bgz",
+                "header_file": "gs://gcp-public-data--gnomad/resources/grch38/dbsnp/dbsnp_b156_grch38_all_GCF_000001405.40_20221116.vcf.header",
+                "force_bgz": True,
+                "contig_recoding": DBSNP_B154_CHR_CONTIG_RECODING,
```
Comment (jkgoodrich) [inline gnomad/resources/grch38/reference_data.py:167]: Just confirming that B156 uses the same recoding as B154?

Comment (KoalaQin) [reply gnomad/resources/grch38/reference_data.py:167]: Yes, I checked for chr1, the file is too big, that I didn't get to all the lines.

File: gnomad/resources/grch38/reference_data.py (hunk 1 above)
Comment (KoalaQin) [reply gnomad/resources/grch38/reference_data.py]: I removed this comment.

**Review by jkgoodrich — APPROVED** [review]
lgtm!

**Pull request merged**